map/search_mark.cpp

```diff
@@ -151,37 +163,48 @@ class SearchMarkTypeChecker
       {{"shop", "beverages"},        SearchMarkType::ShopAlcohol},
       {{"shop", "wine"},             SearchMarkType::ShopAlcohol},
       {{"shop", "butcher"},          SearchMarkType::ShopButcher},
-      {{"shop", "boutique"},         SearchMarkType::ShopClothes},
```

Comment:
Hm, I've just checked wiki and this type is deprecated and should be replaced with shop=clothes.
ToDo: Replace it in the classifier later?